tests/components/homekit/test_aidmanager.py

```diff
@@ -533,91 +529,91 @@ async def test_aid_generation_no_unique_ids_handles_collision(
+        "light.light27": 471811742,
+        "light.light28": 387923647,
+        "light.light29": 371146028,
+        "light.light3": 318355466,
+        "light.light30": 421331790,
+        "light.light31": 438109409,
+        "light.light32": 387776552,
+        "light.light33": 404554171,
+        "light.light34": 488442266,
+        "light.light35": 505219885,
+        "light.light36": 454887028,
+        "light.light37": 471664647,
+        "light.light38": 287110838,
+        "light.light39": 303888457,
+        "light.light4": 234467371,
+        "light.light40": 454048385,
+        "light.light41": 437270766,
+        "light.light42": 420493147,
+        "light.light43": 403715528,
+        "light.light44": 521158861,
+        "light.light45": 504381242,
+        "light.light46": 487603623,
+        "light.light47": 470826004,
+        "light.light48": 319827433,
+        "light.light49": 303049814,
+        "light.light5": 217689752,
+        "light.light50": 353235576,
+        "light.light51": 370013195,
+        "light.light52": 386790814,
+        "light.light53": 403568433,
+        "light.light54": 420346052,
+        "light.light55": 437123671,
+        "light.light56": 453901290,
+        "light.light57": 470678909,
+        "light.light58": 219014624,
+        "light.light59": 235792243,
+        "light.light6": 268022609,
+        "light.light60": 2266325427,
+        "light.light61": 2249547808,
+        "light.light62": 2299880665,
+        "light.light63": 2283103046,
+        "light.light64": 2333435903,
+        "light.light65": 2316658284,
+        "light.light66": 2366991141,
+        "light.light67": 2350213522,
+        "light.light68": 2400546379,
+        "light.light69": 2383768760,
+        "light.light7": 251244990,
+        "light.light70": 554861194,
+        "light.light71": 571638813,
+        "light.light72": 521305956,
+        "light.light73": 538083575,
+        "light.light74": 487750718,
+        "light.light75": 504528337,
+        "light.light76": 454195480,
+        "light.light77": 470973099,
+        "light.light78": 420640242,
+        "light.light79": 437417861,
+        "light.light8": 167356895,
+        "light.light80": 2735113021,
+        "light.light81": 2718335402,
+        "light.light82": 2701557783,
+        "light.light83": 2684780164,
+        "light.light84": 2668002545,
+        "light.light85": 2651224926,
+        "light.light86": 2634447307,
+        "light.light87": 2617669688,
+        "light.light88": 2600892069,
+        "light.light89": 2584114450,
+        "light.light9": 150579276,
+        "light.light90": 2634300212,
+        "light.light91": 2651077831,
+        "light.light92": 2667855450,
+        "light.light93": 2684633069,
+        "light.light94": 2567189736,
+        "light.light95": 2583967355,
+        "light.light96": 2600744974,
+        "light.light97": 2617522593,
+        "light.light98": 2500079260,
+        "light.light99": 2516856879,
```

Comment:
I was all afraid of the line length of this PR, but it's just fixtures 😄

Reply:
I wanted to make sure I got this one right as the switchover is always going to be painful enough.  Hopefully we can put this problem to bed for good now after we get though 0.111.